example/assets/pages/dynamic.json

```diff
@@ -0,0 +1,284 @@
+                                                "elevation": 4,
+                                                "margin": [
+                                                    0,
+                                                    8
+                                                ],
+                                                "padding": 16
+                                            },
+                                            "child": {
+                                                "type": "column",
+                                                "args": {
+                                                    "mainAxisSize": "min"
+                                                },
+                                                "children": [
+                                                    {
+                                                        "type": "row",
+                                                        "children": [
+                                                            {
+                                                                "type": "flexible",
+                                                                "child": {
+                                                                    "type": "text_form_field",
+                                                                    "id": "form_message",
+                                                                    "args": {
+                                                                        "decoration": {
+                                                                            "hintText": "Write a message",
+                                                                            "labelText": "Message"
+                                                                        },
+                                                                        "validators": [
+                                                                            {
+                                                                                "type": "required"
+                                                                            }
+                                                                        ]
+                                                                    }
+                                                                }
+                                                            }
+                                                        ]
+                                                    }
+                                                ]
+                                            }
+                                        },
+                                        {
+                                            "type": "set_value",
+                                            "args": {
+                                                "dynamicItemsAdd": {
```

Comment:
@jpeiffer Most of the fields needed for `dynamic` function are added through `set_value` and via using variable name in that function call.

It is working fine - but it requires a lot of jumping in the json file and a lot of nesting.
Do you think that it could be passed somehow differently?

Maybe it would be nice to  for example to support map args like:
`##dynamic({type:add,builder:items,target:{index:-1},values:{message:{{form_message}}}})##`